provisioning/tools-versions.yml

```diff
@@ -1,6 +1,7 @@
 asdf_version: "0.10.2"
 azurecli_version: "2.41.0"
 chocolatey_version: "1.2.0"
+chromium_version: "105.0.5195.102"
```

Comment:
one point on this, the linux and windows available versions are currently different, windows is one version newer (although windows has old packages so it works fine).

something to keep in mind for updatecli.

could also just split them to windows / linux versions like in git